src/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -60,16 +60,16 @@ abstract class AbstractHtmlProcessor
     protected $domDocument = null;
 
     /**
-     * @var \DOMXPath
+     * @var \DOMXPath|null
```

Comment:
Could we use `@var ?\DOMXPath`?  It's a matter of style but the 'nullable' syntax is consistent with method signatures recognized by the PHP interpreter.

Reply:
Yes, this seems to work for Psalm. Changed.